### PR TITLE
Creeper Explosion Radius Expression

### PR DESCRIPTION
**Pull request by Wealthyturtle**
### Description

This PR adds the ability to get/set a Creeper entity's explosion radius.

---
**Target Minecraft Versions:**     "Added after 1.12.2", so I used methodExists()
**Requirements:**     None
**Related Issues:**     #1385

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprCreeperExplosionRadius.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+						if (r1 < 0) r1 = 0;
+						c.setExplosionRadius(r1);
+						break;
+					case SET:
+						c.setExplosionRadius(d);
+						break;
+					case DELETE:
+					case RESET:
+						c.setExplosionRadius(0);
+						break;
```
Comment (Blueyescat) [inline src/main/java/ch/njol/skript/expressions/ExprCreeperExplosionRadius.java]: This is not a 'reset', it should set to the default

Comment (Wealthyturtle) [reply src/main/java/ch/njol/skript/expressions/ExprCreeperExplosionRadius.java]: Fixed, thanks!

**Comment by APickledWalrus** [comment]
Might be nice to expand this to other entities that explode (see https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Explosive.html)

**Comment by Wealthyturtle** [comment]
Creepers are not instances of Explosive tho.

**Comment by APickledWalrus** [comment]
yeah... I learned that with my PR from earlier haha. I may PR this but for the other entities.

**Comment by bensku** [comment]
I'd prefer to have this tested, unless it requires online players.

**Pull request closed without merging**